docs/WALKTHROUGH.md

```diff
@@ -0,0 +1,394 @@
+# Service Catalog Demonstration Walkthrough
+
+This document outlines the basic features of the service catalog by walking
+through a basic use case.
+
+## Step 0 - Prerequisites
+
+### Starting Kubernetes with DNS
+
```

Comment:
> If you are using hack/local-up-cluster.sh, ensure ...

Isn't this not widely used anymore? Perhaps even deprecated? It seems minikube has supplanted it.

I worry here that if we give specific instructions for all of the many ways someone could deploy k8s, we'll be here for a while.